packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -251,6 +251,46 @@ describe("Runtime", () => {
 				assert.strictEqual(containerRuntime.flushMode, FlushMode.Immediate);
 			});
 
+			it("Process empty batch", async () => {
+				const containerRuntime = await ContainerRuntime.loadRuntime({
+					context: getMockContext({
+						"Fluid.Container.enableOfflineLoad": true,
+					}) as IContainerContext,
+					registryEntries: [],
+					existing: false,
+					runtimeOptions: {
+						flushMode: FlushMode.TurnBased,
+					},
+					provideEntryPoint: mockProvideEntryPoint,
+				});
+				let callsToEnsure = 0;
+				(containerRuntime as any).ensureNoDataModelChanges = () => {
+					callsToEnsure++;
+				};
+				changeConnectionState(containerRuntime, false, mockClientId);
+
+				// Not connected, so nothing is submitted on flush - just queued in PendingStateManager
+				submitDataStoreOp(containerRuntime, "1", "test", { emptyBatch: true });
+				(containerRuntime as any).flush();
+				changeConnectionState(containerRuntime, true, mockClientId);
```

Comment:
What's the purpose of these lines? I think they're not needed but may be missing something.

Reply:
Ah, it's getting the `emptyBatch` marker into PSM, otherwise it will assert.